js/main.js

```diff
@@ -1,6 +1,47 @@
-
 // so that we can use anywhere, will be used for pretty much everything
 let storage = window.localStorage;
+let usernameForm = document.querySelector("#usernameForm")
```

Comment:
This is a little tricky situation, I 100% understand why it is a "global" variable _but_ it does not really fit. It is not used (will be used) across all/most widgets, only the greetings widget. 

This is fine because I don't see a better or easier way to do this. Or do you have any idea? We should probably, in the future when we add more widgets, separate them into their own js files, can already imagine the headache of implementing the 15th widget into the same file.

Reply:
Yeah, I used it a lot so I though it was better to have it global instead of declaring it all the time. But as you said, we should split the code up before we get to many things in one file

Reply:
Can't comment on the other comment so I do it here. But you are right haha They do the same things so I should move it out :+1: